packages/likert-choice/configure/src/choice-configuation.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import {withStyles} from 'material-ui/styles';
```

Comment:
please upgrade to `@material-ui/core` in `package.json` and use the correct import `import Foo from '@material-ui/core/Foo';`